lib/sanitizeBudget.js

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable no-underscore-dangle */
+const precision = 100; // round converted value to the nearest 100
+
+// Common conversions
+const staticMapping = {
+  ECU: 'EUR',
+  FF: 'FRF',
+  DM: 'DEM',
+};
+
+// Fixed euro conversion rates
+// https://www.ecb.europa.eu/euro/intro/html/index.en.html
+const fixedRates = {
+  BEF: 40.3399,
+  DEM: 1.95583,
+  EEK: 15.6466,
+  IEP: 0.787564,
+  GRD: 340.75,
+  ESP: 166.386,
+  CYP: 0.585274,
+  FRF: 6.55957,
+  ITL: 1936.27,
+  LVL: 0.702804,
+  LTL: 3.4528,
+  LUF: 40.3399,
+  MTL: 0.4293,
+  NLG: 2.20371,
+  ATS: 13.7603,
+  PTE: 200.482,
+  SIT: 239.64,
+  SKK: 30.126,
+  FIM: 5.94573,
+};
+
+const sanitizeValue = value => {
```

Comment:
Sharing some example inputs I tried, not sure whether the results are what the sanitization would aim for.
```
> sanitizeValue(01)
1
> sanitizeValue(012)
10
> sanitizeValue(0,123)
0
> sanitizeValue('123 456')
0
```